cohorts/functions.py

```diff
@@ -214,6 +221,44 @@ def count_filter_fn(filterable_effect, **kwargs):
     filterable_effect_function=lambda filterable_effect: (
         isinstance(filterable_effect.effect, FrameShift)))
 
+frameshift_indel_count = count_effects_function_builder(
+    "frameshift_indel_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_indel and
+        isinstance(filterable_effect.effect, FrameShift)))
+
+exonic_frameshift_indel_count = count_effects_function_builder(
+    "exonic_frameshift_indel_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_indel and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_deletion_count = count_effects_function_builder(
+    "exonic_frameshift_deletion_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_deletion and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_insertion_count = count_effects_function_builder(
+    "exonic_frameshift_insertion_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_insertion and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_snv_count = count_effects_function_builder(
```

Comment:
This was a question Alex asked as well. I can say, we saw several in our recent analyses.

Theoretically, snvs can cause a new start-site (stop-gain), modify splice sites (not sure if this would be counted by varcode), cause a stop-loss (which apparently leads to no transcription), or can be stop-gain. In practice I've seen a variety of varcode classes, but am working on functions which would make these easier to inspect using cohorts.